### PR TITLE
Add account charts for locale es_AR.

### PR DESCRIPTION
**Pull request by FrankB21**
I am in the process of creating account charts for Argentina. The PR is a Draft for now, I will mark it as ready for review when all translations are completed.

Translated:
- [x] acctchrt_brokerage.gnucash-xea
- [x] acctchrt_business.gnucash-xea
- [x] acctchrt_carloan.gnucash-xea
- [x] acctchrt_cdmoneymkt.gnucash-xea
- [x] acctchrt_checkbook.gnucash-xea
- [x] acctchrt_childcare.gnucash-xea
- [x] acctchrt_common.gnucash-xea
- [x] acctchrt_currency.gnucash-xea
- [x] acctchrt_eduloan.gnucash-xea
- [x] acctchrt_fixedassets.gnucash-xea
- [x] acctchrt_homeloan.gnucash-xea
- [x] acctchrt_homeown.gnucash-xea
- [x] acctchrt_otherloan.gnucash-xea
- [x] acctchrt_renter.gnucash-xea
- [x] acctchrt_retiremt.gnucash-xea
- [x] acctchrt_spouseinc.gnucash-xea
- [x] acctchrt_spouseretire.gnucash-xea

### CODE REVIEW

**Comment by fellen** [comment]
About commit [081e017](https://github.com/Gnucash/gnucash/pull/1351/commits/081e017e1b056992013be0638e9cb87b08713b39): You got the docs wrong
"**Transactions** involving these accounts should not be added…"
You should create the _accounts_ —usually only one per used currency— but to record _transactions_ into them use <q>post invoice</q> ot <q>process payment</q>.

**Review by fellen — COMMENTED** [review]
From a first glimpse, I see only common and business have translations.
Perhaps you should set the draft flag on this pull request until you think, it is ready.

Tip: I would use a diff editor like kdiff3 and "merge" the translations from the respective files from es_ES and es_MX into es_AR if appropriate.

**Comment by FrankB21** [comment]
> From a first glimpse, I see only common and business have translations. Perhaps you should set the draft flag on this pull request until you think, it is ready.

Yes, the PR is set as Draft already, I have only translated the two most common files (business and common) and will continue with the rest over the next couple days.

> About commit [081e017](https://github.com/Gnucash/gnucash/pull/1351/commits/081e017e1b056992013be0638e9cb87b08713b39): You got the docs wrong

Ahh you're right. Still, I had created quite a few more accounts than just one per currency. I'll fix that right away.

Thanks for the other tips, too.

**Comment by fellen** [comment]
Syntax seems OK.

Probably unstranslated:
Both retirement files. (Savings for) retirement is like investment,  but with different taxation in the US. If you do not have such rules, just drop the files.

Edu-loan: Some have to take out a credit to study at the university and later have to pay it back.

accounts: Cable, Hobbies 

Missing placeholders:
Activo (circulante, fijo), Egresos, Entretenimiento, Inversiones, Cuenta de inversión, Impuestos*, Intereses, Rodados, Seguros, Servicios, Ingresos por intereses, Intereses Ganados, Pasivo, Préstamos

From where did you copy? I would like to fix also the old files.

Usually you should _not merge_ maint into your branch, but _rebase_ your branch on top of maint and the 'git push -f' (force push).

Can you also squash all of your commits into one?

**Comment by fellen** [comment]
oh, wrong button ;-)

**Review by fellen — CHANGES_REQUESTED** [review]
See above

**Comment by FrankB21** [comment]
> accounts: Cable, Hobbies

Cable is the same word in spanish as in english, just a different pronounciation. Hobby is an accepted loan word in spanish, that's why it's not translated.

**Comment by FrankB21** [comment]
I think it's ready. The only thing left is to squash the commits but before that I want to know if you want me to do anything with my mistake of merging maint ([91ede59](https://github.com/Gnucash/gnucash/pull/1351/commits/91ede59cb1595c3c2c8a4c356298a319b3eb4cb1)).

File: data/accounts/es_AR/acctchrt_business.gnucash-xea [hunk 1]
```diff
@@ -0,0 +1,844 @@
...
+    <cmdty:id>ARS</cmdty:id>
+  </act:commodity>
+  <act:commodity-scu>100</act:commodity-scu>
+  <act:description>Gastos de Administración</act:description>
+  <act:slots>
+    <slot>
+      <slot:key>placeholder</slot:key>
+      <slot:value type="string">true</slot:value>
+    </slot>
+  </act:slots>
```
Comment (fellen) [inline data/accounts/es_AR/acctchrt_business.gnucash-xea:610]: There are no sub accounts. Was this intended?

**Review by fellen — COMMENTED** [review]
The rest seems OK to me.

If you are expirienced enough with git,
rebase on current maint head,
and squash them into one commit.
If not I can do it for you.

The difference of optical nature: if you do it, this PR will end as `merged`, if I do it, I will note `squashed into commit ###` and mark it as `closed`.

File: data/accounts/es_AR/acctchrt_business.gnucash-xea (hunk 1 above)
Comment (FrankB21) [reply data/accounts/es_AR/acctchrt_business.gnucash-xea:610]: Fixed!

Comment (fellen) [reply data/accounts/es_AR/acctchrt_business.gnucash-xea]: Note: I have updated https://wiki.gnucash.org/wiki/Account_Hierarchy_Template#Testing_in_Gnucash

**Review by fellen — COMMENTED** [review]
Finally you want perhaps run 'git commit --amend' and shorten the commit message to imprtant parts.

**Review by fellen — COMMENTED** [review]
OK, thanks!

**Pull request merged**